docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -170,24 +170,41 @@ under ``localhost:7990``.
    -  Add capabilities menu → Capability type ``JDK`` → insert ``JDK``
       as JDK label → insert ``/usr/lib/jvm/java-15-oracle`` as Java home.
 
-9. Generate a personal access token for Bamboo.
-   While username and password can still be used as a fallback, this option is already marked as deprecated and
-   will be removed in the future.
+9. Generate a personal access token
 
-   - Log in as the admin user and go to Bamboo -> Profile (top right corner) -> Personal access tokens -> Create token
+   While username and password can still be used as a fallback, this option is already marked as deprecated and will
+   be removed in the future.
 
-       .. figure:: bamboo-bitbucket-jira/bamboo-create-token.png
-          :align: center
+   9.1 Personal access token for Bamboo.
+   
+      - Log in as the admin user and go to Bamboo -> Profile (top right corner) -> Personal access tokens -> Create token
 
-   - Copy the generated token to your ``application-local.yml``:
+          .. figure:: bamboo-bitbucket-jira/bamboo-create-token.png
+             :align: center
 
-   .. code:: yaml
+      - Copy the generated token to your ``application-local.yml``:
```

Comment:
The rest of the documentation mentions the file `application-artemis.yml`, we should also do this here!

Reply:
```suggestion
      - Copy the generated token to the file ``application-artemis.yml``:
```